O/object_store_ffi/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+
+name = "object_store_ffi"
+version = v"0.1.0"
+
+sources = [
+    # https://github.com/RelationalAI/object_store_ffi/commit/c03a55e65512136c91b6f74787191d34ce2c48b3
+    GitSource("https://github.com/RelationalAI/object_store_ffi.git", "c03a55e65512136c91b6f74787191d34ce2c48b3")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/
```

Comment:
Presumably you want to enter the directory object_store_ffi?

Reply:
:lightbulb: 
```suggestion
cd ${WORKSPACE}/srcdir/object_store_ffi/
```